### PR TITLE
Add TestSecurityPolicyBasicTraffic e2e test

### PR DESCRIPTION
**Pull request by zhengxiexie**

### CODE REVIEW

File: test/e2e/framework.go [hunk 1]
```diff
@@ -448,21 +467,24 @@ func (data *TestData) runCommandFromPod(podNamespace string, podName string, con
...
 	}); err != nil {
+		log.Printf("Error running command: %v, stdout: %s, stderr: %s", err, stdoutB.String(), stderrB.String())
 		return stdoutB.String(), stderrB.String(), err
 	}
+	log.Printf("Running command, stdout: %s, stderr: %s", stdoutB.String(), stderrB.String())
 	return stdoutB.String(), stderrB.String(), nil
 }
 
-func (data *TestData) runPingCommandFromTestPod(podName string, targetPodIPs *PodIPs, count int) error {
+func (data *TestData) runPingCommandFromTestPod(testNamespace string, podName string, targetPodIPs *PodIPs, count int) error {
```
Comment (TaoZou1) [inline test/e2e/framework.go:486]: testNamespace -> podNamespace ?

File: test/e2e/framework.go [hunk 2]
```diff
@@ -479,7 +501,7 @@ func (data *TestData) runPingCommandFromTestPod(podName string, targetPodIPs *Po
 	return nil
 }
 
-func (data *TestData) runNetcatCommandFromTestPod(podName string, server string, port int) error {
+func (data *TestData) runNetcatCommandFromTestPod(testNamespace string, podName string, server string, port int) error {
```
Comment (TaoZou1) [inline test/e2e/framework.go:504]: testNamespace -> podNamespace ?

File: test/e2e/framework.go [hunk 3]
```diff
@@ -330,6 +327,27 @@ func (data *TestData) deletePodAndWait(timeout time.Duration, name string, ns st
 
 type PodCondition func(*corev1.Pod) (bool, error)
 
+// securityPolicyWaitFor polls the K8s apiServer until the specified SecurityPolicy is in the "True" state (or until
+// the provided timeout expires).
+func (data *TestData) securityPolicyWaitFor(timeout time.Duration, name, namespace string) error {
```
Comment (TaoZou1) [inline test/e2e/framework.go:332]: securityPolicyWaitFor -> waitForSecurityPolicyReady ?

File: test/e2e/framework.go (hunk 1 above)
Comment (zhengxiexie) [reply test/e2e/framework.go:486]: ok

File: test/e2e/framework.go (hunk 2 above)
Comment (zhengxiexie) [reply test/e2e/framework.go:504]: ok

File: test/e2e/framework.go (hunk 3 above)
Comment (zhengxiexie) [reply test/e2e/framework.go:332]: ok

**Review by TaoZou1 — APPROVED** [review]

**Pull request merged**